Home/Home 3.2/Triangle.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+class Triangle {
```

Comment:
The same comments related to input validation and so on

Reply:
Refactored the code in [90ac7f0](https://github.com/YaroslavKSE/Professinal-Programming/pull/10/commits/90ac7f03ae60329ab88db27d18a6b1b2a16da546)